proj/src/sub1/test/b.ml

```diff
@@ -2,5 +2,9 @@
    Tests for Sub1.B
 *)
 
+let test_string () =
+  Alcotest.(check @@ neg string) "foo is not bar" "foo" "bar"
```

Comment:
Not sure I like the use of `@@`. I wouldn't expect beginners to know what it means.